packages/example/src/pages/guides/configuration.mdx

```diff
@@ -170,6 +170,7 @@ To add a link to the bottom of each page that points to the current page source
   - `page`: "Configuration" (default)
   - `site`: "Gatsby Theme Carbon"
   - `append`: "Gatsby Theme Carbon – Configuration"
+  - `prepend`: "Configuration - Gatsby Theme Carbon"
```

Comment:
Looks good. Should we update line 168 to be "Here are the _four_ options..."?

> - `titleType` – pick between three formats for the `<title>` element for your site. Here are the three options using this page as an example:

Reply:
oh yea, good catch